Assets/Scripts/MiniGames/Song/SongGameController.cs

```diff
@@ -55,40 +130,92 @@ void Start()
         startPlaying = false;
     }
 
-    // Update is called once per frame
-    void Update()
+	void lose_game(float c) 
+	{
+		if (!setmessage) 
+			{
+			if (c > .6f) { failure.text = lose_message[Random.Range(0, lose_message.Count - 1)]; }
+			else { failure.text = lose_message_badly[Random.Range(0, lose_message_badly.Count - 1)]; }
+		}
+		//failure
+		
+		int x = Random.Range(1, 4);
+		endGameState = true;
+		lose.interactable = true;
+		//lose.alpha += Time.deltaTime; cant use here
+		setmessage = true;
+	}
+
+	void win_game(float score) {
+		if (!setmessage) 
+			{
+			if (score > .60f) {
+				clout = (int)(15 * score / 100);
```

Comment:
clout is only used in this function, so you could just make it a local variable. eg
````
var clout = 0;
if(!setmessage) {
    ...
}
````

Reply:
done